core/lib/src/config/config.rs

```diff
@@ -409,6 +425,11 @@ impl Config {
             None => launch_meta_!("IP header: {}", "disabled".paint(VAL))
         }
 
+        match self.proxy_proto_header {
+            Some(ref name) => launch_meta_!("Proxy-Proto header: {}", name.paint(VAL)),
```

Comment:
Slight simplification:

```suggestion
        match self.proxy_proto_header.as_ref() {
            Some(name) => launch_meta_!("Proxy-Proto header: {}", name.paint(VAL)),
```